src/commands/matchmaking.ts

```diff
@@ -1,19 +1,49 @@
 import { Command } from "../lib/Command"
 import config from "../lib/config";
 import { CommandArg, CommandExecuteParameters } from "../types"
-//import bot from "../app" //Need bot/discord client to find the role.
-const options: CommandArg[] = []
+import getPlayerCount from "../utils/getPlayerCount";
+const options: CommandArg[] = [];
 
 const cooldown = 60;
 
 const matchmaking = async (options: CommandExecuteParameters) => {
   const { type, message, interaction } = options
   const isSlash = type === "interaction"
-  var response = `<@&${config.matchmaking_role_id}>!`;
-  return isSlash ? interaction?.reply((response + ` Pinged by <@${interaction.user.id}>`)) : message?.reply((response + ` Pinged by <@${message.author.id}>`));
-  //.displayName: https://old.discordjs.dev/#/docs/discord.js/main/class/User?scrollTo=displayName
-  // Apparently doesn't exist acording to ts :shrug:
-  //Discord's interaction is .user and message is .author and is quite annoying.
+  var matchmakingPing = `<@&${config.matchmaking_role_id}>!`; // Default response
+
+  const playerCount = await getPlayerCount(true)
+
+  if (!playerCount || playerCount == null) { //If "", 0, etc || null return default response
+    return isSlash ? 
+    interaction?.reply((matchmakingPing + ` Pinged by <@${interaction.user.id}>`)) 
+    : message?.reply((matchmakingPing + ` Pinged by <@${message.author.id}>`));
+  }
+  
+  //If we are here, we have not had an issue yet.
+
+  var playerCountText:string = "";
+  if (typeof playerCount === "string") {
+    playerCountText = `There are currently **${playerCount}.**`
+  } else {
+    var operativesText = `There are currently **${playerCount.Online_players} operatives online.**`;
+    var lobbiesText = `There are **${playerCount.Open_lobbies} open lobbies.**`;
+    var activeGamesText = `**There are ${playerCount.Active_games} active games.**`;
+    // Converts the each type (Online_players, Open_lobbies, Active_games) to number to get a more response. 
+  
+    // EG: "There are currently **1 operatives online.**" -> "**There is currently 1 operative online.**"
+    if (Number(playerCount.Online_players) == 1) operativesText = "**There is currently 1 operative online.**"; // If (1 == 1), should also = 1.
+    // EG: "There are **1 open lobbies**" -> "There is **1 open lobby**"...
+    if (Number(playerCount.Open_lobbies) == 1) lobbiesText = "There is **1 open lobby.**";
+    // EG: ..."1 active games" -> "1 active game."
+    if (Number(playerCount.Active_games) == 1) activeGamesText = "There is **1 active game.**"
+    //Removing the spaces creates left aligned text.
```

Comment:
overall looks good. As a shortcut you can do 
```ts
const operativesText = `**There is currently ${Online_players} ${Number(Online_players) >= 1 ? "operatives" : "operative} online.**`
```
, but not really a big dfiference

Reply:
Not gonna change it so it is more readable and easier to fix in case of a mistake.